src/profile/ostonya-profile.js

```diff
@@ -0,0 +1,53 @@
+async function getOstonya() {
+    const ostonyaApiUrl = "http://localhost:4000/students?slug=ostonya-thomas";
+    try {
+        const response = await fetch(ostonyaApiUrl, {
+            method: "GET"
+        });
+        const data = await response.json();
+        return data;
+    } catch (error) {
+        return console.error(error);
+    }
+}
+async function getNasaBackground() {
+  const nasaApiUrl = "https://api.nasa.gov/planetary/apod?api_key=DEMO_KEY&date=2023-02-02";
```

Comment:
The requirements ask that you get the picture for your `dob` using last year as the year value. What if this function takes in a date parameter?... 👀 

```suggestion
  const nasaApiUrl = `https://api.nasa.gov/planetary/apod?api_key=DEMO_KEY&date=${date}`;
```